ci-operator/templates/openshift/installer/cluster-launch-installer-libvirt-e2e.yaml

```diff
@@ -248,6 +248,21 @@ objects:
           --zone "${GOOGLE_COMPUTE_ZONE}" \
           packer@"${INSTANCE_PREFIX}" \
           --command 'export OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE="${RELEASE_IMAGE_LATEST}" && timeout 4800 bash -ce "create-cluster nested 2>&1 | tee output"' | tee /tmp/artifacts/output-e2e-libvirt
+        echo 'Collect all the info about clusteroperators'
+        LD_PRELOAD=/usr/lib64/libnss_wrapper.so gcloud compute --project "${GOOGLE_PROJECT_ID}" ssh \
+          --zone "${GOOGLE_COMPUTE_ZONE}" \
+          packer@"${INSTANCE_PREFIX}" \
+          --command 'export KUBECONFIG=/home/$USER/clusters/nested/auth/kubeconfig && bash -ce "oc get co | tee output"' | tee /tmp/artifacts/output-e2e-libvirt
+        echo 'Extract openshift-must-gather from the image and put it to executable path'
+        LD_PRELOAD=/usr/lib64/libnss_wrapper.so gcloud compute --project "${GOOGLE_PROJECT_ID}" ssh \
+          --zone "${GOOGLE_COMPUTE_ZONE}" \
+          packer@"${INSTANCE_PREFIX}" \
+          --command 'mkdir -p /tmp/must-gather && oc image extract quay.io/openshift/origin-must-gather:v4.0.0 --path /usr/bin/openshift-must-gather:/tmp/must-gather && chmod +x /tmp/must-gather/openshift-must-gather'
```

Comment:
Instead of hard-coding `quay.io/openshift/origin-must-gather:v4.0.0`, can we extract the pullspec referenced by `RELEASE_IMAGE_LATEST` using [`--image-for`][1]?  At least until we get a `must-gather` target [here][2].

[1]: https://github.com/openshift/origin/blob/1c938ecfe24e4794d57c3363114821e570c241e3/pkg/oc/cli/admin/release/info.go#L76
[2]: https://github.com/openshift/origin/blob/1c938ecfe24e4794d57c3363114821e570c241e3/pkg/oc/cli/admin/release/extract_tools.go#L58

Reply:
@wking Done 👍